lib/committee/parameter_converter.rb

```diff
@@ -0,0 +1,44 @@
+module Committee
+  class ParameterConverter
+    def initialize(params, schema)
+      @params = params
+      @schema = schema
+    end
+
+    def convert!
+      coerced = {}
+      @schema.properties.each do |k, s|
+        original_val = @params[k]
+
+        unless original_val.nil?
+          s.type.each do |to_type|
+            case to_type
+            when "string"
```

Comment:
Would you mind if we moved both the check on type and format into the same place? I think it's a little clearer as to what's going on. i.e.

``` ruby
if to_type == "string" && s.format == "date-time"
  coerced_val = parse_date_time(original_val)
```